Makefile.in

```diff
@@ -267,6 +267,24 @@ runtest:
 	  ARCH=`grep '^ARCH=' ocaml/Makefile.config | cut -d'=' -f2` \
 	  $(dune) runtest --root=. --profile=release --build-dir=_build2
 
+# Only needed for running the test tools by hand; runtest will take care of
+# building them using Dune
+#
+# CR lmaurer: stage2 should be a dependency here, but I'm wary because
+# there may be a reason runtest doesn't depend on stage2.
+.PHONY: test-tools
+test-tools:
+	PATH=$(stage1_prefix)/bin:$$PATH \
+	  ARCH=`grep '^ARCH=' ocaml/Makefile.config | cut -d'=' -f2` \
+	  $(dune) build @middle_end/flambda2/tests/tools/all \
+	    --root=. --profile=release --build-dir=_build2
+
+.PHONY: promote
+promote:
```

Comment:
I'm not sure if this should be called just "promote".  There is a "promote" target in the upstream testsuite too which we should expose here at some stage...